h2o-core/src/main/java/hex/grid/HyperSpaceWalker.java

```diff
@@ -517,9 +520,9 @@ public MP nextModelParameters(Model previousModel) {
               double timeleft = this.time_remaining_secs();
               if (timeleft > 0)  {
                 if (params._max_runtime_secs > 0) {
-                  params._max_runtime_secs = (long) floor(min(params._max_runtime_secs, timeleft));
+                  params._max_runtime_secs = min(params._max_runtime_secs, timeleft);
```

Comment:
removed rounding that can lead to have small timeout converted to 0 (no timeout): tried to keep double all the way to avoid those annoying rounding issue